content/blog/18-19/g2_blogpost.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "A Gateway to collaboration"
+created_at: 26-02-2018
+description: Het ProjectenProject
+author: Arne Bertrand
+---
+"Ik wil meedoen met een zeus project, hoe doe?"
+
+Het is een vraag die vervelend vaak voorkomt en verbazend moeilijk te 
+beantwoorden is. De keuze van het project op zich is vaak al lastig. 
+Welke technologie wil ik gebruiken, welke projecten zijn nog relevant,
+welk project heeft het meeste hulp nodig, zijn stuk voor stuk aspecten
+die meespelen en elkaar vaak tegenspreken.
```

Comment:
Tegenspreken? In welke zin?

Reply:
relevant != interessant. Weinig mensen willen docs schrijven of oude code refactoren, hoewel dit vaak het meest nuttige is

Reply:
Ahzo. Ik vind t een rare formulering. Misschien "... die meespelen en zelden in harmonie zijn met elkaar." of "... die meespelen en zelden een duidelijk antwoord hebben."?

Opnieuw, t is maar een voorstel :) Het hier bij laten kan ook gerust.

Reply:
hmm nee kzie toch liever tegenspreken :) maar bedankt ;)